crates/cli-support/src/wit/incoming.rs

```diff
@@ -155,6 +155,8 @@ impl InstructionBuilder<'_, '_> {
 
             // Largely synthetic and can't show up
             Descriptor::ClampedU8 => unreachable!(),
+
+            Descriptor::NonNull => self.number(AdapterType::NonNull, WasmVT::I32),
```

Comment:
It might be a good idea to add some kind of comment mentioning that this is unused right now due to only `Option<NonNull<T>>` implementing `FromWasmAbi`.

Reply:
I removed it and added an `unimplemented!()` instead.